paasta_tools/cli/cmds/logs.py

```diff
@@ -383,9 +419,42 @@ def print_logs_by_offset(self, service, line_count, offset, levels, components,
         raise NotImplementedError("print_logs_by_offset is not implemented")
 
 
+ScribeComponentStreamInfo = namedtuple('ScribeComponentStreamInfo', 'per_cluster, stream_name_fn, filter_fn, parse_fn')
+
+
 @register_log_reader('scribereader')
 class ScribeLogReader(LogReader):
     SUPPORTS_TAILING = True
+    SUPPORTS_LINE_COUNT = True
+    SUPPORTS_TIME = True
+
+    COMPONENT_STREAM_INFO = {
+        'default': ScribeComponentStreamInfo(
+            per_cluster=False,
+            stream_name_fn=get_log_name_for_service,
+            filter_fn=paasta_log_line_passes_filter, parse_fn=None
```

Comment:
I generally prefer to either do one param per line or break at 120 characters and not mix. Can you move parse_fn to the next line?